### ISSUE TITLE
Scheme not found

### ISSUE THREAD

**Issue by MilesWuCode** [nuxt-community/auth-module#664]
v5
Scheme facebook wasn't found!
Scheme google wasn't found!

**Comment by JoaoPedroAS51** [comment]
Hi @MilesWuCode! Can you show me your auth config?

**Comment by MilesWuCode** [comment]
i used this git: demo folder

**Comment by MilesWuCode** [comment]
```json
export default {
  mode: 'universal',
  /*
   ** Headers of the page
   */
  head: {
    title: process.env.npm_package_name || '',
    meta: [
      { charset: 'utf-8' },
      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
      {
        hid: 'description',
        name: 'description',
        content: process.env.npm_package_description || ''
      }
    ],
    link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }]
  },
  /*
   ** Customize the progress-bar color
   */
  loading: { color: '#fff' },
  /*
   ** Global CSS
   */
  css: [],
  /*
   ** Plugins to load before mounting the App
   */
  plugins: [],
  /*
   ** Nuxt.js dev-modules
   */
  buildModules: [
    // Doc: https://github.com/nuxt-community/eslint-module
    '@nuxtjs/eslint-module',
    // Doc: https://github.com/nuxt-community/stylelint-module
    '@nuxtjs/stylelint-module',
    // Doc: https://github.com/nuxt-community/nuxt-tailwindcss
    '@nuxtjs/tailwindcss'
  ],
  /*
   ** Nuxt.js modules
   */
  modules: [
    // Doc: https://axios.nuxtjs.org/usage
    '@nuxtjs/axios',
    '@nuxtjs/pwa',
    // Doc: https://github.com/nuxt-community/dotenv-module
    '@nuxtjs/dotenv',
    '@nuxtjs/auth-next'
  ],
  /*
   ** Axios module configuration
   ** See https://axios.nuxtjs.org/options
   */
  axios: {},
  /*
   ** Build configuration
   */
  build: {
    /*
     ** You can extend webpack config here
     */
    extend(config, ctx) {}
  },
  auth:{
    strategies:{
      facebook: {
        endpoints: {
          userInfo: 'https://graph.facebook.com/v2.12/me?fields=about,name,picture{url},email,birthday'
        },
        clientId: '1671464192946675',
        scope: ['public_profile', 'email', 'user_birthday']
      },
      google: {
        clientId:
          '956748748298-kr2t08kdbjq3ke18m3vkl6k843mra1cg.apps.googleusercontent.com'
      }
    }
  }
}

```

**Comment by radudiaconu0** [comment]
@JoaoPedroAS51  i get this error too on every scheme

**Comment by radudiaconu0** [comment]
if (builtInSchemes.includes(scheme)) {
        return '~auth/schemes/' + scheme;
    }
here you forogt to put the folder....

**Comment by MilesWuCode** [comment]
@JoaoPedroAS51 
node : 14.1.0
nuxt (not typescript) +  @nuxtjs/auth-next
and
```
      google: {
        clientId:
          '956748748298-kr2t08kdbjq3ke18m3vkl6k843mra1cg.apps.googleusercontent.com'
      }
```
show error : Scheme google wasn't found!
---
---
```
      google: {
        provider: 'google',
        scheme: 'oauth2',
        clientId:
          '956748748298-kr2t08kdbjq3ke18m3vkl6k843mra1cg.apps.googleusercontent.com'
      }
```
can run build
but...(brower) login button click not happend

**Comment by MilesWuCode** [comment]
> if (builtInSchemes.includes(scheme)) {
> return '~auth/schemes/' + scheme;
> }
> here you forogt to put the folder....

Try it
provider: 'google',
scheme: 'oauth2',
but .... on brower not working !?
# is TypeScript on your project ?

**Comment by radudiaconu0** [comment]
@JoaoPedroAS51 i thik the proble is you put providers into subfolders?
 const builtInProvider = resolve(__dirname, '../providers', provider);
    for (const _paths of [provider, builtInProvider]) {
      for (const _path of _paths)
        try {
            const m = _nuxt.resolver.requireModule(_path);
            return m.default || m;
        }
        catch (e) {
            // TODO: Check if e.code is not file not found, throw an error (can be parse error)
            // Ignore
        }
    }

**Comment by JoaoPedroAS51** [comment]
Thank you for the report guys! We are going to fix this ASAP :)

**Comment by JoaoPedroAS51** [comment]
Fixed in commit 6ff915c :)
